composer.json

```diff
@@ -15,20 +15,20 @@
         }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony/routing": "^3.4 || ^4.3",
-        "symfony/http-kernel": "^3.4 || ^4.3",
+        "php": "^7.2",
+        "symfony/routing": "^3.4 || ^4.3 || ^5.0",
```

Comment:
i suggest that we restrict oursevles to `^4.4 || ^5.0`. 4.3 is fading out, and 3.4 users can use version 2 of this library.

Reply:
Good 👍